README.md

```diff
@@ -1,25 +1,34 @@
-# Role Name
+# Git repsoitory deploy
 
-This role copys the contents of the inventory hostname to the webserver
+This role clones git repository to the given directory on the target server.
 
 
 ## Requirements
 
-you need the nginx role
-
+None 
 
 ## Role Variables
 
+| Variable name | Required | Description |
+| -------- | -------- | -------- |
+| `static_web_folder_to_copy`     | :heavy_check_mark:     | The variable contains every pair of `src` and `to` that is needed to deploy a git repository     |
```

Comment:
From the example I gather that `src` may be a git repo address. Is https also allowed?
Please describe at least `src`.